src/pages/saksbehandling/revurdering/formue/FormuevilkårOppsummering.tsx

```diff
@@ -0,0 +1,95 @@
+import classNames from 'classnames';
+import { Element, Ingress, Normaltekst } from 'nav-frontend-typografi';
+import React from 'react';
+
+import * as DateUtils from '~lib/dateUtils';
+import { useI18n } from '~lib/hooks';
+import { FormueVilkår, VurderingsperiodeFormue } from '~types/grunnlagsdataOgVilkårsvurderinger/formue/Formuevilkår';
+
+import messages from './formue-nb';
+import styles from './formue.module.less';
+
+const FormuevilkårOppsummering = (props: { gjeldendeFormue: FormueVilkår; brukesForOppsummering?: boolean }) => {
+    const intl = useI18n({ messages });
+
+    return (
+        <div>
+            {!props.brukesForOppsummering && (
+                <div className={styles.eksisterendeVedtakTittelContainer}>
+                    <Ingress>{intl.formatMessage({ id: 'eksisterende.vedtakinfo.tittel' })}</Ingress>
+                </div>
+            )}
+            <ul>
+                {props.gjeldendeFormue.vurderinger.map((vurdering) => (
+                    <li key={vurdering.id}>
+                        <Formuevurdering vurdering={vurdering} />
+                    </li>
+                ))}
+            </ul>
+        </div>
+    );
+};
+
+export const Formuevurdering = (props: { vurdering: VurderingsperiodeFormue }) => {
+    const { vurdering } = props;
```

Comment:
Hvis man først skal destrukturere så kan det like gjerne gjøres sånn
```suggestion
export const Formuevurdering = ({ vurdering }: { vurdering: VurderingsperiodeFormue }) => {
```